lib/ansible/modules/cloud/amazon/lambda.py

```diff
@@ -228,25 +230,57 @@ def sha256sum(filename):
     return hex_digest
 
 
+# There is a PR open to merge fail_json_aws this into the standard module code;
+# see https://github.com/ansible/ansible/pull/23882
```

Comment:
This PR was closed. The relevant PR became https://github.com/ansible/ansible/pull/25780 and is now merged. This block should be deleted.